django/thunderstore/community/templates/community/communities_list.html

```diff
@@ -0,0 +1,215 @@
+                        </a>
+                    </div>
+                </div>
+            </a>
+        {% endfor %}
+        </div>
+        {% if is_paginated %}
+            {% include "repository/includes/pagination.html" with page_obj=page_obj paginator=paginator %}
+        {% endif %}
+    {% else %}
+        <li class="my-4">No communities found :(</li>
+    {% endif %}
+
+    <div class="row mb-2 ml-2 mr-2">
+        <div class="col-12">
+            <div class="d-flex flex-row justify-content-end">
+                <p class="text-dark mb-2 ml-4" >Missing a game you'd like to see? Message us on <a href="https://discord.gg/UWpWhjZken">Discord</a>.</p>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="flex-d flex-row mod-man-banner" id="mod-manager">
+    <img class="mod-manager-pic float-left mr-5 mt-3" src="{% static 'icon.png' %}">
+    <div class="row justify-content-right text-wrap">
+        <h1 class="align-items-end">Get Thunderstore mod manager <span class="ts-text-highlight">today</span></h1>
+    </div>
+    <div class="row justify-content-right">
+        <p class="mt-2 text-left text-muted">The perfect application for easily installing mods to your games.</p>
+        <a class="btn btn-primary ml-0 ts-primary-button" href="https://www.overwolf.com/app/Thunderstore-Thunderstore_Mod_Manager">Download</a>
+    </div>
+</div>
+
+<div class="row justify-content-center">
+    <div class="col-12 mt-4 text-center">
+    <div class="navbar navbar-expand-sm navbar-dark rounded community-list-footer">
+        <ul class="navbar-nav nav-fill w-100">
+            <span class="footer-item ml-5 mr-2">
+                <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="18" height="18" viewBox="0 0 18 18"><defs><clipPath id="a"><rect width="18" height="18" transform="translate(441 1062)" fill="#fff" stroke="#707070" stroke-width="1"/></clipPath></defs><g transform="translate(-441 -1062)" clip-path="url(#a)"><path d="M16.855,2.689a.675.675,0,0,0-.664.684v1.4A8.093,8.093,0,1,0,9.569,18.8a.675.675,0,0,0,.2-1.334A6.745,6.745,0,1,1,15.359,5.847H13.718a.675.675,0,1,0,0,1.349h3.167l.049,0,.061-.009.052-.012.013,0,.006,0,.023-.009.04-.015q.028-.012.055-.026l.005,0q.027-.015.053-.032l.005,0q.025-.017.048-.037l.005,0q.023-.02.045-.041l0,0q.021-.022.04-.046l0-.005.011-.015q.015-.019.028-.04a.674.674,0,0,0,.033-.059q.014-.027.026-.055l0-.006q.011-.029.019-.058l0-.006q.008-.03.014-.06V6.63q0-.031.007-.061V6.554q0-.016,0-.033V3.373a.675.675,0,0,0-.685-.684Zm-3.353,9a1.807,1.807,0,0,0-1.809,1.8v5.4a1.8,1.8,0,0,0,1.8,1.8l5.394-.008a1.8,1.8,0,0,0,1.8-1.8l0-5.387a1.8,1.8,0,0,0-1.8-1.8Zm5.387,2.241a.45.45,0,0,1,.318.768l-3.6,3.6a.45.45,0,0,1-.636,0l-1.8-1.8a.45.45,0,0,1,.636-.636l1.481,1.481,3.28-3.28A.448.448,0,0,1,18.89,13.935Z" transform="translate(438.306 1059.311)" fill="#fff"/></g></svg>
```

Comment:
I'm not a fan of these megalong lines here. Could the SVG be served as separate files (caching if they're reused elsewhere on the site), or indented here, oslt?

Reply:
Again I have to say that my brain power is limited on solving this one out as requested. Maybe @gkurck has some neat way of putting all the svgs to one file and calling them by ID or something?

Reply:
There are a few viable options on how to use SVG's in html:

- Inline. Able to control appearance with CSS: ✅ Yes
- External file with `<img>`. Able to control appearance with CSS: ❌ No
- As a symbol which is referenced inline elsewhere, [example](https://codepen.io/gravija/pen/OJvMWoG). Able to control appearance with CSS: ✅ Yes